packages/okta-vue/test/e2e/harness/test/e2e/specs/test.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
       .assert.urlContains('/protected')
       .assert.elementPresent('#logout-button')
       .assert.containsText('.protected', 'Protected!')
+      .pause(2000)
```

Comment:
Was this flakey before?

Reply:
It failed a few times locally. This fix helped